src/BlazorUI/Bit.BlazorUI/Styles/Fluent/forced-colors.fluent.scss

```
@@ -0,0 +1,45 @@
+        --bit-clr-pri-hover: LinkText;
+        --bit-clr-pri-active: LinkText;
+        --bit-clr-pri-text: Canvas;
+        --bit-clr-err: CanvasText;
+        --bit-clr-suc: CanvasText;
+        --bit-clr-wrn: CanvasText;
+        --bit-shd-cal: none;
+        --bit-shd-sm: none;
+        --bit-shd-nm: none;
+        --bit-shd-md: none;
+        --bit-shd-lg: none;
+        // Focus must remain visible in High Contrast — use system Highlight color and a single solid ring.
```

Stylelint (17.11.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)